AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -142,7 +144,11 @@ private var runtimeTargets: [Target] {
             name: "AWSSDKChecksums",
             dependencies: [.crt, .smithy, .clientRuntime, .smithyChecksumsAPI, .smithyChecksums, .smithyHTTPAPI],
             path: "Sources/Core/AWSSDKChecksums/Sources"
-        )
+        ),
+        .target(
+            name: "AWSSDKPartitions",
+            path: "Sources/Core/AWSSDKPartitions/Sources"
+        ),
```

Comment:
Creates a new target named `AWSSDKPartitions` to contain the partitions data.

Partitions has its own separate target to help keep generated code separate from hand-maintained runtime code.